README.md

```diff
@@ -53,14 +53,14 @@ All additional compatibility is only best effort, or happens to still/already be
 #### Compatibility matrix
 At most 5 kube-state-metrics releases will be recorded below.
 
-| kube-state-metrics | client-go  | **Kubernetes 1.9** | **Kubernetes 1.10** | **Kubernetes 1.11** | **Kubernetes 1.12** |
-|--------------------|------------|--------------------|---------------------|---------------------|---------------------|
-| **v1.2.0**         |  v6.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **v1.3.0**         |  v6.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **v1.3.1**         |  v6.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **v1.4.0**         |  v8.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **v1.5.0**         |  v8.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
-| **master**         |  v8.0.0    |         ✓          |         ✓           |         ✓           |         ✓           |
+| kube-state-metrics | client-go  | **Kubernetes 1.10** | **Kubernetes 1.11** | **Kubernetes 1.12** | **Kubernetes 1.13** | **Kubernetes 1.14** |
+|--------------------|------------|---------------------|---------------------|---------------------|---------------------|---------------------|
+| **v1.2.0**         |  v6.0.0    |         ✓           |         ✓           |         ✓           |         -           |         -           |
+| **v1.3.1**         |  v6.0.0    |         ✓           |         ✓           |         ✓           |         -           |         -           |
+| **v1.4.0**         |  v8.0.0    |         ✓           |         ✓           |         ✓           |         -           |         -           |
+| **v1.5.0**         |  v8.0.0    |         ✓           |         ✓           |         ✓           |         -           |         -           |
+| **v1.6.0**         |  v11.0.0   |         ✓           |         ✓           |         ✓           |         -           |         -           |
```

Comment:
there should be all checkmarks on this line

Reply:
Fixed.